easyaudit/signals/request_signals.py

```diff
@@ -69,14 +70,12 @@ def request_started_handler(sender, **kwargs):
         session_cookie_name = settings.SESSION_COOKIE_NAME
         if session_cookie_name in cookie:
             session_id = cookie[session_cookie_name].value
-
-            try:
-                session = Session.objects.get(session_key=session_id)
-            except Session.DoesNotExist:
-                session = None
+            engine = import_module(settings.SESSION_ENGINE)
```

Comment:
I don't particularly mind this flow, but is there a less brutal way of doing this for users that do not require as much flexibility in the call flow? Maybe a new settings flag?
Also, is there harm in resolving the `engine` in the module scope, as opposed to the function scope? (Perhaps there is, like a django init thing, but wanted to ask just in case there was no danger.)

Reply:
Are you referring to the use of `import_module`? It's the recommended way in the [Django docs](https://docs.djangoproject.com/en/dev/topics/http/sessions/#using-sessions-out-of-views). If the user is retrieved from `request.user` this would not be required at all actually (see my comment in #165).

I agree that the session store should be retrieved once instead of each request. Maybe move it to the top of the module where `audit_logger` is defined?